Tasks/VsTest/vstest.ts

```diff
@@ -1549,21 +1549,27 @@ function getVSTestConsole15Path(): string {
 function locateVSVersion(version: string): Q.Promise<ExecutabaleInfo> {
     let deferred = Q.defer<ExecutabaleInfo>();
     let vsVersion: number = parseFloat(version);
-    
-    if (isNaN(vsVersion) || vsVersion == 15) {
-        // latest
-        tl.debug('Searching for latest Visual Studio');
-        let vstestconsole15Path = getVSTestConsole15Path();
-        if (vstestconsole15Path) {
-            deferred.resolve({version: 15, location: vstestconsole15Path});
+
+    if(vstestLocationMethod.toLowerCase() !== 'location') {    
```

Comment:
Also, whichever definition is hitting this now will also hit this after S113 upgrade. Old definitions which defaulted to 14.0 will continue to default to that